web/src/main/webapp/WEB-INF/conf/application.properties

```diff
@@ -5,7 +5,17 @@ db.password=
 db.insertSchema=true
 db.insertDemodata=true
 
+delivery.price=5
+
 validation.id.empty=Id cannot be empty
```

Comment:
Usually such messages are placed in separate localized resource bundles, just hint for future